northstar/src/runtime/state.rs

```diff
@@ -543,7 +542,7 @@ impl State {
             .chain(once(format!("{}={}", ENV_CONTAINER, container)))
             .chain(once(format!("{}={}", ENV_NAME, container.name())))
             .chain(once(format!("{}={}", ENV_VERSION, container.version())))
-            .map(|s| NonNulString::try_from(s).unwrap())
+            .map(|s| unsafe { NonNulString::from_string_unchecked(s) })
```

Comment:
there are other places where we do a `NonNulString::try_from(...)?` we probably can also use `unsafe` if this will never cause a problem with `std::env`

Reply:
> there are other places where we do a `NonNulString::try_from(...)?`

No. That's the only one and we're sure that it's fine. All the env values (and the keys) come from container names and versions which are also `NonNulString`.